dist/docker/redhat/build_docker.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash -ex
+
+#
+# Copyright (C) 2021-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+
+trap 'echo "error $? in $0 line $LINENO"' ERR
+
+product="$(<build/SCYLLA-PRODUCT-FILE)"
+version="$(sed 's/-/~/' <build/SCYLLA-VERSION-FILE)"
+release="$(<build/SCYLLA-RELEASE-FILE)"
+
+mode="release"
+
+arch="$(uname -m)"
+
+
+print_usage() {
+    echo "usage: $0 [--mode mode]"
+    exit 1
+}
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        --mode)
+            mode="$2"
+            shift 2
+            ;;
+        *)
+            print_usage
+            ;;
+    esac
+done
+
+# CMake puts deb packages in build/$<CONFIG>, so translate $mode
+# to $<CONFIG> if build.ninja is located under build/.
+config=$mode
+if [ -f build/build.ninja ]; then
+   case $mode in
+       release)
+           config=RelWithDebInfo
+           ;;
+       dev)
+           config=Dev
+           ;;
+       debug)
+           config=Debug
+           ;;
+       *)
+           echo "unsupported mode: ${mode}"
+           exit 1
+           ;;
+   esac
+fi
+
+bcp() { buildah copy "$container" "$@"; }
+run() { buildah run "$container" "$@"; }
+bconfig() { buildah config "$@" "$container"; }
+
+container="$(buildah from docker.io/redhat/ubi9-minimal:latest)"
+
+packages=(
+    "build/dist/$config/redhat/RPMS/$arch/${product}-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-server-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-conf-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-kernel-conf-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-node-exporter-$version-$release.$arch.rpm"
+    "tools/cqlsh/build/redhat/RPMS/$arch/$product-cqlsh-$version-$release.$arch.rpm"
+    "tools/python3/build/redhat/RPMS/$arch/$product-python3-$version-$release.$arch.rpm"
+)
+
+bcp "${packages[@]}" packages/
+
+bcp dist/docker/scyllasetup.py /scyllasetup.py
+bcp dist/docker/commandlineparser.py /commandlineparser.py
+bcp dist/docker/docker-entrypoint.py /docker-entrypoint.py
+
+bcp dist/docker/scylla_bashrc /scylla_bashrc
+
+bcp LICENSE-ScyllaDB-Source-Available.md /licenses/
+
+run microdnf clean all
+run microdnf update
+run microdnf -y upgrade
+run microdnf -y install kmod
```

Comment:
Why do we need kmod?

Reply:
We added `Requires: kmod` on https://github.com/scylladb/scylladb/commit/66022214425b6ce0bd89462822f4ef8575ab018b to set modprobe parameter, but seems like we already dropped the code but only dependency is remain.
Probably we can drop the dependency safely